com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java

```diff
@@ -73,7 +75,12 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
                     context.sendEvent(Events.OutputEvent.createConsoleOutput(warnMessage));
                 }
             }
+        } catch (VMDisconnectedException e) {
```

Comment:
This exception seems like a common one for all the handlers. Can we handle them at the framework level?